imagingTable.Rmd

```diff
@@ -29,8 +29,8 @@ library(magrittr)
 
 ```{r,echo=FALSE,results='hide',,message=FALSE, warning=FALSE}
 # Load file for Imaging Table
-generalData <- read_csv(file = here::here("data/ictr_table_imaging.csv"))
-generalData
+generalData <- read_csv(file = here::here("data/ictr_table_imaging_new.csv")) 
```

Comment:
what is the difference between ictr_table_imaging.csv and ictr_table_imaginag_new.csv?

Also the letters got rearranged here - we could rename it using git mv oldname newname

Reply:
The _ictr_table_imaging.csv_ is the original file. The _new is the csv file with the new entries. Overwriting the original file did not work since it kept adding the same row (new tool information) every time it compiled the Rmd file.